rc/filetype/groovy.kak

```diff
@@ -0,0 +1,91 @@
+# Adapted from the file created by Daniel Lewan TeddyDD
+
+hook global BufCreate "(.+\.(groovy|gvy|gy|gsh))|.+[Jj]enkinsfile.*" %{
+  set-option buffer filetype groovy
+  set-option buffer comment_line '//'
+  set-option buffer comment_block_begin '/*'
+  set-option buffer comment_block_begin '*/'
+}
+
+hook global WinSetOption filetype=groovy %{
+  require-module groovy
+  hook window ModeChange pop:insert:.* -group groovy-trim-indent %{ try %{ execute-keys -draft <a-x>s^\h+$<ret>d } }
```

Comment:
the `<a-x>` needs to be changed to `x` to accomodate the breaking change (sorry for the long delay)

```suggestion
  hook window ModeChange pop:insert:.* -group groovy-trim-indent %{ try %{ execute-keys -draft xs^\h+$<ret>d } }
```

same below